crates/sui-indexer/src/test_utils.rs

```diff
@@ -63,30 +71,15 @@ pub async fn start_test_indexer_impl(
         format!("postgres://postgres:{pw}@{pg_host}:{pg_port}")
     });
 
-    // dynamically set ports instead of all to 9000
-    let base_port = rpc_url
-        .chars()
-        .rev()
-        .take(4)
-        .collect::<String>()
-        .chars()
-        .rev()
-        .collect::<String>()
-        .parse::<u16>()
-        .unwrap();
-
-    // Set connection timeout for tests to 1 second
-    let mut pool_config = PgConnectionPoolConfig::default();
-    pool_config.set_connection_timeout(Duration::from_secs(1));
+    let pool_config = PgConnectionPoolConfig::default();
```

Comment:
These seem unrelated, but I assume the default now does the connection timeout to 1s.

Reply:
rather, now we can divorce ourselves from having to game around with the connection timeout.